### PR TITLE
Documentation: so2: Add survey section in the filesystems-part1 lab

### PR DESCRIPTION
**Pull request by laurentiustefan97**
Add a section in the filesystems-part1 lab (lab8) for the students to
complete the teaching survey created by the computer department.

Signed-off-by: Laurentiu Stefan <steflaurentiu@gmail.com>

### CODE REVIEW

File: Documentation/teaching/so2/lab8-filesystems-part1.rst [hunk 1]
```diff
@@ -4,3 +4,10 @@ SO2 Lab 08 - File system drivers (Part 1)
 
 .. include:: ../labs/filesystems_part1.rst
     :start-line: 4
+    :end-line: 447
+
+.. important::
+   In order to have a better understanding of what we do well and we can do better, what factors affect your implication in teaching, extracurricular but also professional activities, we ask you to complete `this survey <https://forms.office.com/r/SqBF2kfzk5>`_. The survey is a short one, having answers with check marks, with an estimated completion time of 3-5 minutes. Obviously, we will send you the analysis of the survey and use it to improve the teaching activities.
```
Comment (valighita) [inline Documentation/teaching/so2/lab8-filesystems-part1.rst:10]: Please split this in multiple lines.

Comment (laurentiustefan97) [reply Documentation/teaching/so2/lab8-filesystems-part1.rst]: Fixed it.

**Review by valighita — APPROVED** [review]

File: Documentation/teaching/so2/lab8-filesystems-part1.rst [hunk 2]
```diff
@@ -4,3 +4,16 @@ SO2 Lab 08 - File system drivers (Part 1)
 
 .. include:: ../labs/filesystems_part1.rst
     :start-line: 4
+    :end-before: [SURVEY-LABEL]
+
+.. important::
+   In order to have a better understanding of what we do well and we can do
+   better, what factors affect your implication in teaching, extracurricular
```
Comment (razvand) [inline Documentation/teaching/so2/lab8-filesystems-part1.rst:11]:
```suggestion
   better, what factors affect your involvement in teaching, extracurricular
```

File: Documentation/teaching/so2/lab8-filesystems-part1.rst [hunk 3]
```diff
@@ -4,3 +4,16 @@ SO2 Lab 08 - File system drivers (Part 1)
...
+   but also professional activities, we ask you to complete `this survey
```
Comment (razvand) [inline Documentation/teaching/so2/lab8-filesystems-part1.rst:12]:
```suggestion
   and professional activities, we ask you to complete `this survey
```

File: Documentation/teaching/so2/lab8-filesystems-part1.rst [hunk 4]
```diff
@@ -4,3 +4,16 @@ SO2 Lab 08 - File system drivers (Part 1)
...
+   <https://forms.office.com/r/SqBF2kfzk5>`_. The survey is a short one,
+   having answers with check marks, with an estimated completion time of
+   3-5 minutes. Obviously, we will send you the analysis of the survey and
```
Comment (razvand) [inline Documentation/teaching/so2/lab8-filesystems-part1.rst:15]:
```suggestion
   5-7 minutes. Obviously, we will send you the analysis of the survey and
```

File: Documentation/teaching/so2/lab8-filesystems-part1.rst [hunk 5]
```diff
@@ -4,3 +4,16 @@ SO2 Lab 08 - File system drivers (Part 1)
...
+   use it to improve the teaching activities.
```
Comment (razvand) [inline Documentation/teaching/so2/lab8-filesystems-part1.rst:16]:
```suggestion
   use it to improve our activities in the faculty.
```

**Review by razvand — CHANGES_REQUESTED** [review]
I added feedback as suggestions.

**Comment by laurentiustefan97** [comment]
> I added feedback as suggestions.

@razvand I addressed the feedback.

**Review by razvand — APPROVED** [review]
All good.

**Pull request merged**